### PR TITLE
Make Header Use Consistent

### LINKED ISSUE #118: Make the header use consistent.
The library specifies two prefixes that define headers:

- `X-Xmidt-` prefix found [here](https://github.com/xmidt-org/wrp-go/blob/e7c67a5ebf305f867014706a34cc02ab39b30a82/wrphttp/headers.go#L31).
- `X-Midt-` prefix found [here](https://github.com/xmidt-org/wrp-go/blob/e7c67a5ebf305f867014706a34cc02ab39b30a82/wrphttp/headers.go#L244).

Presently the library consistently outputs with the prefix `X-Xmidt-`.

This is confusing and error prone, so we want to change the library to accept either prefix form (`X-Xmidt-` and `X-Midt-`) and always output `X-Xmidt-`.

The exported constants for the headers will also only be of the prefix form `X-Xmidt-` to reduce confusion as well.

This preserves backwards compatibility.

### PR DESCRIPTION
**Pull request by renaz6**
Closes #118 
Make header use consistent by ensuring that wrp-go accepts both header prefixes:
`X-Xmidt-*` and `X-Midt-*`

### CODE REVIEW

File: wrphttp/headers.go [hunk 1]
```diff
@@ -57,14 +80,17 @@ var (
 func getMessageType(h http.Header) wrp.MessageType {
 	value := h.Get(MessageTypeHeader)
 	if len(value) == 0 {
-		panic(errMissingMessageTypeHeader)
+		// check alternative header prefix
+		value = h.Get(msgTypeHeader)
+		if len(value) == 0 {
```
Comment (denopink) [inline wrphttp/headers.go:85]:
```suggestion
		if value == "" {
```

File: wrphttp/headers.go [hunk 2]
```diff
@@ -57,14 +80,17 @@ var (
 func getMessageType(h http.Header) wrp.MessageType {
 	value := h.Get(MessageTypeHeader)
 	if len(value) == 0 {
```
Comment (denopink) [inline wrphttp/headers.go:82]:
```suggestion
	if value == "" {
```

**Review by denopink — APPROVED** [review]
some small suggestions

Otherwise, lgtm 🍻

File: wrphttp/headers.go (hunk 2 above)
Comment (schmidtw) [reply wrphttp/headers.go:82]: The rest of the code around this is also using `len(string) == 0`.  While that seems less than ideal, I think consistency is better in this case.  We can/should go through as a separate PR and change this pattern in the future if we want; but consistently.

Comment (renaz6) [reply wrphttp/headers.go:82]: Yeah, I was just matching what was already there, but I'll update to your suggestion! It's more readable that way

**Review by schmidtw — APPROVED** [review]

File: wrphttp/headers.go (hunk 2 above)
Comment (renaz6) [reply wrphttp/headers.go:82]: As Wes pointed out, the rest of the around this also uses `len(string) == 0` and consistency is better at the moment. We will leave this conditional as it is.

File: wrphttp/headers.go (hunk 1 above)
Comment (renaz6) [reply wrphttp/headers.go:85]: As Wes pointed out, the rest of the around this also uses `len(string) == 0` and consistency is better at the moment. We will leave this conditional as it is.

**Pull request merged**